src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/PersonPersisterTest.kt

```diff
@@ -24,6 +26,8 @@ internal class PersonPersisterTest(
 ) {
   private val personRepositorySpy: PersonRepository = mock { spy(personRepository) }
   private val profileRepositorySpy: ProfileRepository = mock { spy(profileRepository) }
+  private val personCaptor = argumentCaptor<List<Person>>()
```

Comment:
These two lines are a bit of a hack.  Upgrading to the lastest dps spring boot plugin cause a compilier failure use the any() function.